.github/workflows/pr-title.yml

```
@@ -0,0 +1,45 @@
+name: Pull Request Title
+
+on:
+  pull_request:
```

```suggestion
on:
  pull_request:
    types:
      - opened
      - edited
      - reopened
```

Don't you need to declare the types edited, to make it so the CI runs on title change

---

oh good point